README.md

```diff
@@ -5,6 +5,30 @@
 
 > The future is here; let's evenly distribute it!
 
+# Installation
+
+Use without installing:
+
+```
+nix-shell \
+  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update.cachix.org/' \
+  --option trusted-public-keys '
+  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+  nixpkgs-update.cachix.org-1:6y6Z2JdoL3APdu6/+Iy8eZX2ajf09e4EE9SnxSML1W8=
+  ' -p /nix/store/jmsdhi0jqw3i8jd0wbwmcvkvxxmywrr8-nixpkgs-update-0.2.0
+```
+
+Install it:
+
+```
+nix-env \
+  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update.cachix.org/' \
+  --option trusted-public-keys '
+  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+  nixpkgs-update.cachix.org-1:6y6Z2JdoL3APdu6/+Iy8eZX2ajf09e4EE9SnxSML1W8=
+  ' -i /nix/store/jmsdhi0jqw3i8jd0wbwmcvkvxxmywrr8-nixpkgs-update-0.2.0
+```
```

Comment:
I've recently started to make heavy use of `nix run` commands, like:

```
nix run nixpkgs.python3Packages.black -c black  .
```
or
```
nix run nixpkgs.nixpkgs-review -c nixpkgs-review pr 1234
```
or from this very project
```
nix run nixpkgs.cabal2nix -c cabal2nix --shell --hpack . > shell.nix
```

These are really awesome, because you can paste them into IRC/Discord/a github PR comment for someone else to run without messing up their env, making assumptions about their env, or requiring multi-step cmds like with `nix-shell`.  

It'd be cool if we got `nixpkgs-update` streamlined to the point you could do something like:

```
nix run nixpkgs.nixpkgs-update -c update "foo 1.0 2.0"
```
and have it "just work".

Reply:
Yes, I'm not sure nixpkgs-update can be integrated into nixpkgs well, because at least currently, building it requires a specific pinned version of nixpkgs.

Reply:
Maybe once flakes are a thing this will be no problem.

Reply:
This does work, though :man_dancing: 
```
nix run -f https://github.com/ryantm/nixpkgs-update/archive/master.tar.gz -c nixpkgs-update --help 
```